external/storm-redis/src/main/java/org/apache/storm/redis/state/RedisKeyValueState.java

```diff
@@ -266,10 +288,10 @@ public void rollback() {
                 LOG.debug("hmset txidNamespace {}, txIds {}", txidNamespace, txIds);
                 commands.hmset(txidNamespace, txIds);
             }
-            pendingCommit = Collections.emptyMap();
-            pendingPrepare = new ConcurrentHashMap<>();
+            pendingCommit = createPendingCommitMap();
```

Comment:
nit: can be `Collections.emptyNavigableMap()` (java8) or a static final variable initialized with an empty TreeMap